githubscript/tests/test_integration.py

```diff
@@ -0,0 +1,129 @@
+import json
```

Comment:
Future: I'd love for us to do some integration testing in taskgraph in scriptworker-scripts. That would probably involve some low-privilege creds in secrets.

Reply:
We may say the same things with different words: We could even do some end-to-end tests thanks to taskgraph and low-privilege creds. The sw-script gets an actual task that interacts with a staging service.